pom.xml

```diff
@@ -557,6 +570,71 @@
 				</plugins>
 			</build>
 		</profile>
+		
+		<!-- Uses tycho-surefire instead of maven-surefire to run tests that require an Equinox platform -->
+		<profile>
+			<id>standalone-test</id>
+			<activation>
+				<file>
+					<exists>.tests-without-platform</exists>
+				</file>
+			</activation>
+			<dependencies>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-engine</artifactId>
+				</dependency>
+			</dependencies>
+			<build>
+				<plugins>
+					<plugin>
+			    		<groupId>org.eclipse.tycho</groupId>
+					    <artifactId>tycho-surefire-plugin</artifactId>
+					    <executions>	
+					    	<execution>
+					    		<id>default-test</id>
+					    		<phase>none</phase>
+					    	</execution>
+					    </executions>
+		 		   </plugin>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>test</id>
+								<phase>test</phase>
+								<goals>
+									<goal>test</goal>
+								</goals>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		
+		<!-- Uses tycho-surefire instead of maven-surefire to run tests that require a full Eclipse workbench -->
```

Comment:
This documentation does also seem inappropriate :-) 
```suggestion
		<!-- Runs tests with tycho-surefire that require a full Eclipse workbench -->
```

Reply:
Done.